.pylintrc

```diff
@@ -432,7 +432,8 @@ disable=raw-checker-failed,
         use-implicit-booleaness-not-comparison-to-string,
         use-implicit-booleaness-not-comparison-to-zero,
         too-few-public-methods,
-        missing-module-docstring
+        missing-module-docstring,
+        import-error
```

Comment:
Note to reader: we disabled the import-error flag in pylint because it was triggering unnecessarily